tests/actions.go

```diff
@@ -531,9 +531,10 @@ func (oa *operatorActions) CheckTidbClusterStatus(info *TidbClusterConfig) error
 		}
 
 		glog.V(4).Infof("check tidb cluster begin metaSyncFn")
-		if b, err := oa.metaSyncFn(tc); err != nil {
-			return false, err
-		} else if !b && err == nil {
+		if b, err := oa.metaSyncFn(tc); !b && err == nil {
+			return false, nil
+		} else if err != nil {
+			glog.Error(err)
```

Comment:
could you document why this error can be omitted?

Reply:
other sync functions log the error [immediately](https://github.com/pingcap/tidb-operator/blob/31db0ea157d05e0e62d32626b2384cd33cbec772/tests/actions.go#L1064). `metaSyncFn ` return these errors and log them outside of the method.
these may be different coding styles.